ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -381,6 +383,12 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
             DEFAULT_EXT_DIR,
             ConfigDef.Importance.LOW,
             "The path to look for and load extensions such as UDFs from."
+        ).define(
+            KSQL_INTERNAL_TOPIC_REPLICAS_PROPERTY,
+            Type.SHORT,
+            (short) 1,
```

Comment:
I know this would be a change of behavior, but I feel like we should increase the default replication to 2 or 3, or have some guidelines (linked here) for configuring production deployments of KSQL. We don't want a user losing their query data if they worked hard to get them up and running.

Reply:
The tricky thing with defaulting to more than 1 is that you need to have at least as many Kafka brokers as the value of this property. Defaulting to more than 1 means KSQL wouldn't work out of the box with a Kafka cluster with only one broker, which someone could reasonably want to do for testing.

Reply:
Ah that is tricky! I suppose the latter (production guidelines) is the better approach.